### ISSUE TITLE
Expandable Content failing test on `aria-expanded` attribute

### ISSUE THREAD

**Issue by jnicholCU** [CuBoulder/tiamat-theme#614]
Expandable content block throws a critical ADA error for aria elements
Error: `Ensures an element's role supports its ARIA attributes` 
Current ARIA attributes: `aria-expanded="true" aria-controls="EXPANDABLE-CONTENT-ID#"`
